### PR TITLE
UP-4877: Improve ADA in Marketplace for 5.0.0

### PR DESCRIPTION
**Pull request by colcampbe3**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] [message properties][] have been updated with new phrases
-   [x] view conforms with [WCAG 2.0 AA][]

##### Description of change

Ported all changes made in the other UP-4877 pull request to uPortal 5. 

- Table formatting is improved and has headers
- Text colors have more contrast
- Forms have labels
- Lists are in proper list structures
- Menus are keyboard-navigable
- Modals are keyboard-navigable

https://issues.jasig.org/browse/UP-4877

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by drewwills** [comment]
@ChristianMurphy -- Do you have a bit of bandwidth to see this one to the finish line?  It would be good to make 5.0.0 with it.

**Comment by ChristianMurphy** [comment]
@drewwills I'll see what I can do, my bandwidth is limited at the moment.

File: uPortal-webapp/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp [hunk 1]
```diff
@@ -401,14 +407,86 @@
         </div>
         <div class="row">
             <div class="col-xs-12">
-                <h1><spring:message code="rating.and.review.cap" text="Ratings & Reviews"/></h1>
+                <h1>
+                    <spring:message code="rating.and.review.cap" text="Ratings & Reviews"/>
```
Comment (drewwills) [inline uPortal-webapp/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp]: Should this text be `" Ratings &amp; Reviews"`?

File: uPortal-webapp/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp [hunk 2]
```diff
@@ -18,6 +18,7 @@
     under the License.
 
 --%>
+<%@page contentType="text/html;charset=UTF-8"%>
```
Comment (drewwills) [inline uPortal-webapp/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp]: Why was this part needed again?

Does it matter that the output of this JSP will be combined into a bigger page with other markup that (almost certainly) didn't include this declaration?

Comment (ChristianMurphy) [reply uPortal-webapp/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp]: AFAICT the `charset` is being set in the XSLT.
The JSP also needs to know the content type, otherwise it breaks the ☆ symbols into three different icons.

**Comment by asgrant** [comment]
Thanks to everyone for your work on this.
